gap/PolygonalComplexes/graphs.gd

```diff
@@ -850,5 +851,49 @@ DeclareAttribute( "FaceDigraphsGraph", IsPolygonalSurface );
 DeclareAttribute( "FaceNautyGraph", IsPolygonalSurface );
 #! @EndGroup
 
-
-
+# TODO If there are more methods which construct surfaces with a given property, it is useful to have a method AllSimplicialSurfacesByFacesOfEdges which use this method.
+#! @BeginGroup AllSimplicialSurfacesOfGraph 
+#! @Description 
+#! Return all (vertex-faithful) simplicial surfaces, that have <K>digraph</K> as face graph. 
+#! If <K>digraph</K> is not a face graph of a (vertex-faithful) simplicial surface, the empty list is returned.
+#! The parameter <K>vertexfaithful</K> indicates whether only vertex-faithful simplicial surfaces are searched. 
+#! The parameter <K>vertexfaithful</K> is by default false.
+#! <K>digraph</K> must be a cubic, connected, symmetric and simple digraph. The vertices of a simplicial 
+#! surface are certain cycles in the face graph. The method search possible combinations of cycles, 
+#! with the cycles forming the corners of a simplicial surface.
```

Comment:
Instead of
"
The method search possible combinations of cycles, with the cycles forming the corners of a simplicial surface.
"
This method searches possible combinations of cycles, with the cycles corresponding to the vertices of a simplicial surface.